src/main/webapp/app/exam/participate/exercises/exam-exercise-update-highlighter/exam-exercise-update-highlighter.component.scss

```diff
@@ -0,0 +1,17 @@
+#highlightDiffButton {
+    background-color: #fbc107;
+    border: none;
+    padding: 0 10px;
```

Comment:
Could we use bootstrap spacing classes for `highlightDiffButton` instead? 
Documentation: https://getbootstrap.com/docs/5.0/utilities/spacing/
In this case it seems it would be: `py-0` and `px-2`

Reply:
Sure, thank you for the information, this is new for me 👍